organization/controllers.py

```diff
@@ -55,48 +55,54 @@ def organization_retrieve_tweets_from_twitter(organization_we_vote_id, number_to
     twitter_user_manager = TwitterUserManager()
     tweets_saved = 0
     tweets_not_saved = 0
+
     for tweet_json in new_tweets:
         results = twitter_user_manager.update_or_create_tweet(tweet_json)
         if results['success']:
             tweets_saved += 1
         else:
             tweets_not_saved += 1
 
-    # Gets ID, date and tweet text
-    # out_tweets = [[tweet.id_str, tweet.created_at, tweet.text] for tweet in new_tweets]
     return results
```

Comment:
Currently the results returned are the last results from update_or_create_tweet. I would build a results dict like this:
results = {
'success': True,
'status': status,  # If update_or_create_tweet returns a 'success' of False, you could capture the message above with status += update_or_create_tweet_results['status']
'tweets_saved': tweets_saved,
'tweets_not_saved': tweets_not_saved,
}